package.json

```diff
@@ -111,7 +111,7 @@
   ],
   "homepage": "http://github.com/newrelic/node-newrelic",
   "engines": {
-    "node": ">=10.0.0",
+    "node": ">=12.0.0",
     "npm": ">=3.0.0"
```

Comment:
should we bump this to align with npm 6 as that's what is shipped with 12?

Reply:
Seems fine to me since all these changes require a major version bump.